auth-foundation/src/main/java/com/okta/authfoundation/AuthFoundationDefaults.kt

```diff
@@ -71,4 +73,9 @@ object AuthFoundationDefaults {
     /** The default [CookieJar]. By default, it adds a DT cookie for identifying the device.
      * To use the default [CookieJar] in OkHttp, set this to [CookieJar.NO_COOKIES] */
     var cookieJar: CookieJar by NoSetAfterGetWithLazyDefaultFactory { DeviceTokenCookieJar(clock) }
+
+    /** The default wait time until web login flow is cancelled after receiving empty redirect response from web browser.
+     * This can resolve some issues related to SDK picking up incorrect redirect results from the web browser, but it adds
+     * delay to browser redirect during errors if set to non-zero value. */
```

Comment:
```suggestion
    /** The default wait time until the web login flow is cancelled after receiving empty redirect response from the web browser.
     * This can resolve some issues caused by older devices when invalid redirect results are returned from the older browser. When this is set to a non-zero value, it introduces a
     * delay to all redirects when an error is received. */
```